mealie/pkgs/img/minify.py

```diff
@@ -57,24 +65,32 @@ def purge(self, image: Path):
             return
 
         for file in image.parent.glob("*.*"):
-            if file.suffix != WEBP:
+            if file.suffix != WEBP.suffix:
                 file.unlink()
 
 
 class PillowMinifier(ABCMinifier):
     @staticmethod
-    def to_webp(image_file: Path, dest: Path | None = None, quality: int = 100) -> Path:
-        """
-        Converts an image to the webp format in-place. The original image is not
-        removed By default, the quality is set to 100.
-        """
```

Comment:
Doco was removed altogether, is there a particular reason to do that rather than updating it to reflect the new functionality?

Reply:
Ah yeah I was shuffling functions around so I removed that. I can add it back